src/actions/action-types.js

```diff
@@ -15,3 +15,6 @@ export const UPDATE_SEARCH_INPUT = 'UPDATE_SEARCH_INPUT';
 export const FETCH_SEARCH_INDEX = 'FETCH_SEARCH_INDEX';
 export const FETCH_SEARCH_INDEX_SUCCESS = 'FETCH_SEARCH_INDEX_SUCCESS';
 export const FETCH_SEARCH_INDEX_FAILURE = 'FETCH_SEARCH_INDEX_FAILURE';
+export const FETCH_SUBJECTS_REQUEST = 'FETCH_SUBBJECTS_REQUEST';
```

Comment:
You brought up a really good point about me having used fetch/get interchangeably—I think we should switch to using fetch instead of get from here on out since that's more specific. Though that's tech debt we can tackle later instead of in this ticket.

Reply:
I'll make a note of that.